### PR TITLE
April 1 gfg potd solution

### PR DESCRIPTION
**Pull request by AvinashUmrao**
Screenshot 2024-10-30 085622
Screenshot 2024-10-30 085633
Screenshot 2024-10-30 085644
Screenshot 2024-10-30 085653

### CODE REVIEW

File: April 2024/Adding solution for Pairs violating the BST property in cpp April 1
Comment (Ayu-hack) [inline April 2024/Adding solution for Pairs violating the BST property in cpp April 1:1]: please remove this file. you have two added same files

**Comment by AvinashUmrao** [comment]
@Ayu-hack Done

**Comment by Ayu-hack** [comment]
Great Work!
 Merged.
Thank you for contributing .
Feel free to tag me in your Hacktoberfest certificate or badges LinkedIn post! 😄 
👉 Follow my GitHub and connect with me on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02)!

**Comment by Ayu-hack** [comment]
if possible do follow on [GitHub](https://github.com/Ayu-hack) and connect on [LinkedIn](https://www.linkedin.com/in/ayush-tiwari02) as a token of appreciation for the efforts! 🙌

**Pull request merged**